reflect_test.go

```diff
@@ -176,3 +176,23 @@ func TestBaselineUnmarshal(t *testing.T) {
 
 	require.Equal(t, strings.Replace(string(expectedJSON), `\/`, "/", -1), string(actualJSON))
 }
+
+func TestReflect(t *testing.T) {
+	type Values struct {
+		Value1 int    `json:"value1" jsonschema:"minimum=-1"`
+		Value2 int    `json:"value2" jsonschema:"minimum=0"`
+		Value3 int    `json:"value3" jsonschema:"minimum=1"`
+		Value4 int    `json:"value4" jsonschema_extras:"minimum=0"`
+		Value5 string `json:"value5" jsonschema_extras:"format=json"`
+	}
+
+	reflector := &Reflector{}
+	schema := reflector.Reflect(&Values{})
+
+	schemaBytes, err := schema.MarshalJSON()
+	if err != nil {
+		t.Fatalf("err:%v", err)
+	}
+
+	t.Logf("Schema:%v", string(schemaBytes))
```

Comment:
This test doesn't test anything? And can you name it something more appropriate?

Reply:
sure, I have renamed it.  I was thinking of adding https://github.com/xeipuuv/gojsonschema to validate the JSON schema vs data, But it will add one more dependency. so decided to just print the schema alone.

Reply:
That's not necessary, there's already a fixture based testing framework here. This test should be removed and the test for this functionality added to the existing fixture-based testing system.

Apologies for not picking this up initially, but that's how all of the other testing is done, so it should be fairly clear how to proceed.

Reply:
No worries. Thanks for noticing that. I have fixed test.